e2e/utils/strategy/ManageStrategyDriver.ts

```diff
@@ -0,0 +1,37 @@
+import { CreateStrategyTemplate } from './../../utils/strategy/template';
+import { DebugDriver } from './../../utils/DebugDriver';
+import { navigateTo } from './../../utils/operators';
+import { MyStrategyDriver } from './../../utils/strategy/MyStrategyDriver';
+import { Page } from 'playwright-core';
+
+export class ManageStrategyDriver {
+  constructor(private page: Page) {}
+
+  async createStrategy(config: CreateStrategyTemplate) {
+    const debug = new DebugDriver(this.page);
+    await debug.createStrategy(config);
+    await navigateTo(this.page, '/');
+    const myStrategies = new MyStrategyDriver(this.page);
+    return await myStrategies.getStrategy(1);
+  }
+
+  async waitForEditPage(type: 'deposit' | 'withdraw' | 'renew') {
+    await this.page.waitForURL(`/strategies/edit?type=${type}`, {
+      timeout: 10_000,
+    });
+  }
+
+  async fillBudget(
+    type: 'deposit' | 'withdraw',
+    order: 'buy' | 'sell',
+    budget: string | number
+  ) {
+    await this.page
```

Comment:
I would rather use `return` instead of await : 
- You don't need `async` anymore
- It returns the result in case you need it in the test suite
- You will need to await in the test case anyway
```suggestion
    return this.page
```